lib/Varien/Data/Collection/Db.php

```diff
@@ -586,11 +586,14 @@ public function load($printQuery = false, $logQuery = false)
 
         if (is_array($data)) {
             foreach ($data as $row) {
+                /** @var Mage_Core_Model_Abstract $item */
```

Comment:
`$item` can be `Varien_Object`, if this is the case `$item->setOrigData();` is not applicable,

Reply:
> `$item` can be `Varien_Object`, if this is the case `$item->setOrigData();` is not applicable,

`Varien_Object` has `setOrigData` method

Reply:
My bad.